### PR TITLE
Fixed comments not displaying in readonly view

### LINKED ISSUE #706: Bug: Comments don't appear in read-only shared boards
**Steps to reproduce the behavior:**
1. Add a comment to a card
2. [Share the board](https://www.focalboard.com/guide/user/#sharing-boards)
3. Open the read-only board, then open the card
4. Notice there are no comments

**Expected behavior:**
Comments should appear in shared read-only boards

**Platform:**
 - Focalboard v0.8.0 on Mattermost 5.37 RC1 (but also on previous versions)

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Fixed comments not displaying in readonly view

#### Ticket Link
Fixes #706

### CODE REVIEW

File: webapp/src/components/cardDetail/comment.tsx [hunk 1]
```diff
@@ -60,6 +61,21 @@ const Comment: FC<Props> = (props: Props) => {
                         />
                     </Menu>
                 </MenuWrapper>
+                <div className='comment-date'>{(new Date(comment.createAt)).toLocaleString()}</div>
```
Comment (jespino) [inline webapp/src/components/cardDetail/comment.tsx:64]: This looks like some kind of accidentally included duplication.

**Review by jespino — CHANGES_REQUESTED** [review]
A tiny detail, but otherwise looks good to me

File: webapp/src/components/cardDetail/comment.tsx (hunk 1 above)
Comment (harshilsharma63) [reply webapp/src/components/cardDetail/comment.tsx:64]: Yeah, one component was duplicated. Updating now.

Comment (harshilsharma63) [reply webapp/src/components/cardDetail/comment.tsx]: Donel

**Review by wiggin77 — CHANGES_REQUESTED** [review]
Great work @harshilsharma63 .  Some build errors to address then I'll approve.

**Review by jespino — APPROVED** [review]

**Comment by harshilsharma63** [comment]
> Great work @harshilsharma63 . Some build errors to address then I'll approve.

Thanks!

Working on the fixes. They are due to some changes with redux merge.

**Comment by harshilsharma63** [comment]
@wiggin77 I've fixed the tests now.

**Review by wiggin77 — APPROVED** [review]
LGTM 👍

**Pull request merged**